spec/spec_helper.rb

```diff
@@ -1,8 +1,9 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'govuk/diff/pages'
+require 'pry'
 
 module FixtureHelper
   def self.locate(filename)
```

Comment:
let's rename this method to `load_paths_from` - bit more descriptive, and we're no longer just locating a file path.